Shopist/Shopist/AppDelegate.swift

```diff
@@ -7,29 +7,62 @@
 import UIKit
 import Datadog
 
-fileprivate(set) var logger: Logger!
+internal fileprivate(set) var logger: Logger! // swiftlint:disable:this implicitly_unwrapped_optional
+internal let appConfig = AppConfig(serviceName: "ios-sdk-shopist-app")
+internal var rum: DDRUMMonitor? { Global.rum }
+
+private struct User {
+    let id: String = UUID().uuidString
+    let name: String
+    var email: String {
+        return name.lowercased()
+            .replacingOccurrences(of: " ", with: "@")
+            .appending(".com")
+    }
 
-let appConfig = AppConfig(serviceName: "ios-sdk-shopist-app")
-var rum: DDRUMMonitor? { Global.rum }
+    static let users: [User] = [
+        User(name: "John Doe"),
+        User(name: "Jane Doe"),
+        User(name: "Pat Doe"),
+        User(name: "Sam Doe"),
+        User(name: "Maynard Keenan"),
+        User(name: "Adam Jones"),
+        User(name: "Justin Chancellor"),
+        User(name: "Danny Carey"),
+        User(name: "Karina Round"),
+        User(name: "Martin Lopez"),
+        User(name: "Anneke Giersbergen"),
+        User(name: "Billie Eilish"),
+        User(name: "Cardi B"),
+        User(name: "Nicki Minaj"),
+        User(name: "Beyonce Knowles")
+    ]
+
+    static func any() -> Self {
+        return users.randomElement()! // swiftlint:disable:this force_unwrapping
+    }
+}
 
 @UIApplicationMain
-class AppDelegate: UIResponder, UIApplicationDelegate {
+internal class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-
         // Initialize Datadog SDK
         Datadog.initialize(
             appContext: .init(),
             configuration: Datadog.Configuration
                 .builderUsing(
                     rumApplicationID: appConfig.rumAppID,
-                    clientToken: appConfig.rumClientToken,
+                    clientToken: appConfig.clientToken,
                     environment: "tests"
```

Comment:
`shop.ist` is an illegal service name, apparently `.` is invalid char
i make it `shopist` 👍

Reply:
So we have a different definition of valid `env` between platforms. This must be unified. I created JIRA, so this will be addressed later:
* `RUMM-700 Investigate the `env` name validation in Android RUM and iOS RUM`